WORKSPACE.bazel

```diff
@@ -0,0 +1,3 @@
+workspace(
+    name = "kibana",
```

Comment:
`kibana` or `kbn` pro-cons to either?

Reply:
Sync'd I am fine using `kibana` and changing if we find having shorter would be better.

Reply:
👍